build/m1/Dockerfile-setup

```diff
@@ -25,13 +27,14 @@ RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     dpkg-reconfigure --frontend=noninteractive locales && \
     update-locale LANG=en_US.UTF-8
 
-ENV CASSANDRA_VERSION 3.11.11
+ENV CASSANDRA_VERSION 3.11.13
```

Comment:
Why not `3.11.15`? which seems to be the latest

Reply:
Sure. No preference. I picked it because the other one was using it.